src/Raven.Client/ServerWide/Operations/RestoreBackupOperation.cs

```diff
@@ -13,10 +13,12 @@ namespace Raven.Client.ServerWide.Operations
     public class RestoreBackupOperation : IServerOperation<OperationIdResult>
     {
         private readonly RestoreBackupConfiguration _restoreConfiguration;
+        public string NodeTag;
 
         public RestoreBackupOperation(RestoreBackupConfiguration restoreConfiguration)
         {
             _restoreConfiguration = restoreConfiguration;
+            NodeTag = restoreConfiguration.NodeTag;
```

Comment:
I think that NodeTag should not be part of `RestoreBackupConfiguration`. It contains data that we are sending to the server and this is not related.

Reply:
done